File: CustomerOrder/CustomerOrder.Service/CustomerOrder/Service/DTO/AddComandaDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CustomerOrder.Service.DTO
{
    public class AddComandaDto
    {
        
        public Nullable<int> idClient { get; set; }
        public Nullable<int> idEmployee { get; set; }
        public Nullable<int> idTelefon { get; set; }
        public int idUnicTelefon { get; set; }
        public Nullable<bool> stare { get; set; }
        public String DataDeschidere { get; set; }
        public String DataInchidere { get; set; }
      //  public HashSet<CustomerOrder.Service.DTO.ResponseDefectiuneDto> DefectiuneDtos { get; set; }
    }
}

File: AspNetAngularClient/client/src/app/shared/DTOs/AddComandaDto.ts
import { TelefonDto } from './TelefonDto';

export class AddComandaDto {
  idClient: number;
  idEmployee: number;
  idTelefon: number;
  idUnicTelefon: number;
  stare: boolean;
  DataDeschidere: any;
  DataInchidere: any = null;
}
